social.whalebird.WhalebirdDesktop.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: social.whalebird.WhalebirdDesktop
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+command: start-whalebird.sh
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=/run/dbus:ro
```

Comment:
why do you need this one?

Reply:
I'd like to drop it but for some reason the app uses the system bus and has a weird error in the console without it

Reply:
what kind of weird error?

Reply:
[3:1116/062432.017970:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory

Reply:
You should use `--socket=system-bus --log-system-bus` to find what it tries to access.

Reply:
It looks like it doesn't access anything through the system bus, should I just ignore the error?